autoload/maktaba/syscall.vim

```diff
@@ -135,7 +135,14 @@ endfunction
 function! maktaba#syscall#DoCallForeground(pause) abort dict
   let l:return_data = {}
   if a:pause
-    execute '!' . s:EscapeSpecialChars(self.GetCommand())
+    if has('nvim')
+        let l:cmd = 'noautocmd new | terminal '
```

Comment:
Doesn't neovim have a minimum version that supports the `:terminal` command, too?

Reply:
Looks like `:terminal` first appeared in https://github.com/neovim/neovim/commit/cdedd89d2 which was before the first versioned release - 0.1.0.

Reply:
Gotcha! Could you put a quick comment to mention all significant versions of neovim work, no additional check needed?